src/analyzer/protocol/gssapi/gssapi-protocol.pac

```diff
@@ -1,6 +1,16 @@
-
-type GSSAPI_NEG_TOKEN(is_orig: bool) = record {
+type GSSAPI_SELECT(is_orig: bool) = record {
 	wrapper  : ASN1EncodingMeta;
+	token: case tok_id of {
+		0x0404 -> mic_blob: bytestring &restofdata;
+		0x0504 -> wrap_blob: bytestring &restofdata;
+		default -> neg_token: GSSAPI_NEG_TOKEN(is_orig, is_init);
+	} &requires(is_init) &requires(tok_id);
+} &let {
+	is_init: bool = wrapper.tag == 0x60;
+	tok_id: uint32 = (wrapper.tag << 8) | wrapper.len;
+} &byteorder=littleendian;
```

Comment:
Above mentioned RFC has a lot of big-endian in it for some of the fields in the tokens, but probably only relevant if support was actually added.

Reply:
Yup - and currently I see no reason to actually parse the messages.